servicetests/tests/FleetProvisioning/src/main/java/fleetProvisioning/FleetProvisioning.java

```diff
@@ -0,0 +1,252 @@
+                builder.withLifeCycleEvents(lifecycleEvents);
+                builder.withPort((long)cmdData.input_port);
+                Mqtt5ClientConnectionWrapper connWrapper = new Mqtt5ClientConnectionWrapper();
+                connWrapper.client = builder.build();
+                connWrapper.connection = new MqttClientConnection(connWrapper.client, null);
+                if (connWrapper.connection == null) {
+                    throw new RuntimeException("MQTT5 connection creation failed!");
+                }
+                return connWrapper;
+            } catch (Exception ex) {
+                throw new RuntimeException("Failed to create MQTT311 connection from MQTT5 client", ex);
+            }
+        } else {
+            throw new RuntimeException("Invalid MQTT version specified: " + mqttVersion);
+        }
+    }
+
+    static void onRejectedKeys(ErrorResponse response) {
+        System.out.println("CreateKeysAndCertificate Request rejected, errorCode: " + response.errorCode +
+                ", errorMessage: " + response.errorMessage +
+                ", statusCode: " + response.statusCode);
+        gotResponse.complete(null);
+    }
+
+    static void onRejectedRegister(ErrorResponse response) {
+        System.out.println("RegisterThing Request rejected, errorCode: " + response.errorCode +
+                ", errorMessage: " + response.errorMessage +
+                ", statusCode: " + response.statusCode);
+        gotResponse.complete(null);
+    }
+
+    static void onCreateKeysAndCertificateAccepted(CreateKeysAndCertificateResponse response) {
+        if (response != null) {
+            System.out.println("CreateKeysAndCertificate response certificateId: " + response.certificateId);
+            if (createKeysAndCertificateResponse == null) {
+                createKeysAndCertificateResponse = response;
+            } else {
+                System.out.println("CreateKeysAndCertificate response received after having already gotten a response!");
+            }
+        } else {
+            System.out.println("CreateKeysAndCertificate response is null");
+        }
+        gotResponse.complete(null);
+    }
+
+    static void onRegisterThingAccepted(RegisterThingResponse response) {
+        if (response != null) {
+            System.out.println("RegisterThing response thingName: " + response.thingName);
+            if (registerThingResponse == null) {
+                registerThingResponse = response;
+            } else {
+                System.out.println("RegisterThing response received after having already gotten a response!");
+            }
+        } else {
+            System.out.println("RegisterThing response is null");
+        }
+        gotResponse.complete(null);
+    }
+
+    static void onException(Exception e) {
+        e.printStackTrace();
+        System.out.println("Exception occurred " + e);
+    }
+
+    public static void main(String[] args) {
```

Comment:
Supper Trivial: It would be easier to read if we have `main()` to the end of file or the beginning of file.